interfaces/cython/cantera/reactor.pyx

```diff
@@ -772,13 +773,18 @@ cdef class Valve(FlowDevice):
     :math:`\dot m = 0`. However, an arbitrary function can also be specified,
     such that
 
-    .. math:: \dot m = f(P_1 - P_2)
+    .. math:: \dot m = K_v*f(P_1 - P_2)
 
     where :math:`f` is the arbitrary function that returns the mass flow rate given
-    a single argument, the pressure differential. See the documentation for the
-    `set_valve_coeff` method for an example. Note that it is never possible for
-    the flow to reverse and go from the downstream to the upstream
-    reactor/reservoir through a line containing a `Valve` object.
+    a single argument, the pressure differential. Further, a valve lift curve
+    :math:`l` may be specified using the method `set_valve_lift`, such that
+
+    .. math:: \dot m = K_v*l(t)*f(P_1 - P_2)
```

Comment:
I'd recommend almost any other symbol for the first function here -- `l(t)` looks too much like `1(t)` in most fonts, at least to me.

Reply:
No problem.